### PR TITLE
skeleton setup for universities

### LINKED ISSUE #82: university of washington
https://www.washington.edu/coronavirus/

### LINKED ISSUE #129: stanford
https://twitter.com/ericsson_axel/status/1236144774794493952

### PR DESCRIPTION
**Pull request by deargle**
Edits readme.md to loop over a new `universities` data collection, where each university is its own data file.

data collections with individual files should be easier to maintain with pull requests?

includes cu boulder and uw

closes #82, closes #129

### CODE REVIEW

**Comment by deargle** [comment]
closes #82

**Comment by deargle** [comment]
closes #129

File: _data/universities/cu_boulder.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name: "[University of Colorado-Boulder](https://www.colorado.edu/chancellor/newsletter/chancellor/march-11-2020)"
+status: Remote teaching begins Monday, March 16, through remainder of semester
+last_update: 2020-03-11
```
Comment (drewbrew) [inline _data/universities/cu_boulder.yml]: I like the idea a lot, especially using different files per school.

What do you think about having more data fields in here for closing start date, closing end date, remote start date, grading period end date, and remote end date (if known)?  That might enable you to do some cool stuff with stats.

Comment (deargle) [reply _data/universities/cu_boulder.yml]: My immediate reaction thought to that is that everything is changing _daily_ for uni's expected remote-teaching duration, and that there would be no good mechanism for tracking accurate history for an individual school unless someone were updating daily, sometimes hourly. To illustrate, my uni has been sending out daily updates, with each basically escalating the response. My other thought is that the thought of doings stats with jekyll gives me stomach pains.

But maybe just separate fields for start and end would be do-able.

Comment (drewbrew) [reply _data/universities/cu_boulder.yml]: You're probably right there.

I'm deferring to @phildini on this one. I'm a +1 though.

Comment (deargle) [reply _data/universities/cu_boulder.yml]: I just pushed up a few more fields to each of the data files -- `start`, and `expected_end`. Nothing added to the readme rendering yet though. But they're there for others to follow.

**Comment by phildini** [comment]
So I think this is great but it now needs a rebase since we just moved to index.md as per #563 

Post-rebase I will for sure merge.

**Comment by deargle** [comment]
not a rebase per-se, but I think in spirit I got it right. Formal rebase probably would have meant that I squash and take a patch file of my edits to README.md and then play them onto index.md, but instead I did a git mv of my readme.md onto the current index.md and then cherry-picked out the new readme.md. Good enough?

**Pull request merged**